api/lib/validation/schemaValidator.js

```diff
@@ -93,7 +93,7 @@ var getScalingRuleSchema = function() {
       'metric_type':{ 'type':'string' ,'enum':metricTypeEnum },
       'stat_window_secs':{ 'type':'number','minimum': 60,'maximum': 3600 },
       'breach_duration_secs':{ 'type':'number','minimum': 60,'maximum': 3600 },
-      'threshold':{ 'type':'number','minimum': 1,'maximum': 100 },
```

Comment:
The value could be between 1-100 only when we were looking at usage and not MBs. It could just be any >0 number now.